server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/storage/XodusServerIssueStore.java

```diff
@@ -118,6 +118,7 @@ public class XodusServerIssueStore implements ProjectServerIssueStore {
   private final Path backupFile;
 
   private final Path xodusDbDir;
+  private static final String RULE_DESCRIPTION_CONTEXT_KEY = "ruleDescriptionContextKey";
```

Comment:
I would move the constant with other constants, and rename to `RULE_DESCRIPTION_CONTEXT_KEY_PROPERTY_NAME` for consistency